errors/ka.po

```diff
@@ -1,21 +1,21 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: Squid-3\n"
+"Project-Id-Version: Squid-5\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2017-09-16 17:53+1300\n"
-"PO-Revision-Date: 2018-12-30 03:57+0400\n"
-"Last-Translator: George Machitidze <giomac@gmail.com>\n"
-"Language-Team: Squid Developers <squid-dev@squid-cache.org>\n"
+"PO-Revision-Date: 2023-02-15 15:45+0100\n"
+"Last-Translator: Temuri Doghonadze <temuri.doghonadze@gmail.com>\n"
```

Comment:
@NorwayFun, when this PR is merged, the scripts that update Squid `CONTRIBUTORS` file will want to add your GitHub account name and email to that file. If that is _not_ what you want, please let us know what name/email you want to be used for CONTRIBUTORs entry instead, and we will handle it manually. For example, we can add the name/email that are used on this Last-Translator line instead of that GitHub-based default. Or we can add just your name (without email) if that is your preference. Or nothing at all! Your call.

Reply:
Just add a name, would be enough for everything, I guess.

Reply:
Will do after this PR merges, thank you!

Reply:
Done in #1289.